package.json

```diff
@@ -158,6 +158,7 @@
     "@nx/js": "~16.8.0",
     "@nx/linter": "~16.8.0",
     "@nx/workspace": "~16.8.0",
+    "@o3r/artifactory-tools": "workspace:^",
```

Comment:
is it needed in the workspace scripts?
(if not, maybe you can remove it from `build:tools` as well)